styleguide/style/color.html

```diff
@@ -11,7 +11,87 @@ <h1 class="site-title">Mozilla Styleguide</h1>
     </header>
     #html-include=styleguide/includes/navigation.html
     <main class="main-content" role="main" tabindex="-1">
-      <h2>Feed me Malcolm, feed me!</h2>
+      <h2>Color Scheme</h2>
+      <p>Some description of colours and colour use...</p>
+      <h3>Flat Colours</h3>
+      <ul class="color-swatches">
+        <li>
+          <h4>Mozilla red</h4>
+          <div class="color moz-red"></div>
+          <table class="color-codes">
```

Comment:
You don't say :) I was thinking the exact same thing. The I thought, perhaps not and went with a table with scope set to row.